command.py

```diff
@@ -0,0 +1,27 @@
+from LSP.plugin.core.registry import windows
+from LSP.plugin.core.registry import LspTextCommand
+import sublime
+
+
+class LspIntelephenseReindexWorkspaceCommand(LspTextCommand):
+    """ Re-index the workspace. """
+
+    session_name = __package__
+
+    def run(self, edit: sublime.Edit) -> None:
+        window = self.view.window()
+        if not window:
+            return
+        self._wm = windows.lookup(window)
+        self._stop_server()
+        configs = self._wm.get_config_manager().match_view(self.view)
+        for config in configs:
+            if config.name == self.session_name:
+                config.init_options.set('clearCache', True)
```

Comment:
Won't those changes persist for the live of the ST session? If so then all that dance with stopping the server first, then starting it again seems unnecessary. Could just do the change and then restart the session.

Though if that change persists for session then all newly started sessions will also clear the cache which is not exactly the behavior that I would expect after running that command.